SpringProject/src/it/java/com/gradproject2019/utils/TestUtils.java

```diff
@@ -32,13 +36,21 @@
     public TestRestTemplate restTemplate;
 
     private User user;
+    private User user2;
     private Conference conference;
     public User savedUser;
+    public User userWithManager;
+    public User manager;
     public Conference savedConference;
     private String hashedPassword;
     public Token testToken;
     public HttpHeaders passingHeaders;
+    public HttpHeaders managerPassingHeaders;
     public HttpHeaders failingHeaders;
+    public User savedUserWithManager;
+    public User savedManager;
```

Comment:
Is there a way we can collapse some of these fields? i.e. can `manager` and `savedManager` be the same user? can `user1` or `user2` assume the other roles?